automobile/nature.md

```diff
@@ -97,6 +98,7 @@ position of the trees is randomly generated from this shape.
 the density of three to be generated (in trees per meter square).
 - `type`: Defines the type of threes, in case of `random` type, the forest will be
 mixed.
+- `groundTexture`: Defines the texture of the forest ground.
```

Comment:
I would just add a second sentence to explain that if this field is empty the forest ground is not displayed.